src/v3/src/components/PasswordRequirements/PasswordRequirements.tsx

```diff
@@ -39,26 +37,23 @@ const PasswordRequirements: UISchemaElementComponent<{
 }> = ({ uischema }) => {
   const { data } = useWidgetContext();
   const {
-    fieldKey,
     header,
     id,
     requirements,
     settings,
     userInfo,
     validationDelayMs,
   } = uischema.options as PasswordRequirementsElement['options'];
-  const password = get(data, fieldKey);
+  const password = 'credentials.passcode' in data
+    ? data['credentials.passcode']
+    : data['credentials.newPassword'];
```

Comment:
sorry to nit in these again but i believe this could be even simpler now that `get` isn't needed
```suggestion
  const password = data['credentials.passcode'] ?? data['credentials.newPassword'];
```

or similar